components/infobox/wikis/pubgmobile/infobox_map_custom.lua

```diff
@@ -0,0 +1,131 @@
+---
+-- @Liquipedia
+-- wiki=pubgmobile
+-- page=Module:Infobox/Map/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Template = require('Module:Template')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local Map = Lua.import('Module:Infobox/Map', {requireDevIfEnabled = true})
+
+local Abbreviation = require('Module:Abbreviation')
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+
+local CustomMap = Class.new()
+
+local CustomInjector = Class.new(Injector)
+
+local _args
+
+local GAME = {
+	mobile = '[[Mobile]]',
+	newstate = '[[New State]]',
+	peace = '[[Peacekeeper Elite|Peace Elite]]',
+	bgmi = '[[Battlegrounds Mobile India|BGMI]]',
+}
+
+local MODES = {
+	solo = '[[:Category:Solos Mode Tournaments|Solos]]',
+	duo = '[[:Category:Duos Mode Tournaments|Duos]]',
+	squad = '[[:Category:Squads Mode Tournaments|Squads]]',
+	['2v2'] = '[[:Category:2v2 TDM Tournaments|2v2 TDM]]',
+	['4v4'] = '[[:Category:4v4 TDM Tournaments|4v4 TDM]]',
+	['war mode'] = '[[:Category:War Mode Tournaments|War Mode]]',
+	default = '',
+}
+MODES.solos = MODES.solo
+MODES.duos = MODES.duo
+MODES.squads = MODES.squad
+MODES.tdm = MODES['2v2']
+
+local PERSPECTIVES = {
+	fpp = {'FPP'},
+	tpp = {'TPP'},
+	mixed = {'FPP', 'TPP'},
+}
+PERSPECTIVES.first = PERSPECTIVES.fpp
+PERSPECTIVES.third = PERSPECTIVES.tpp
```

Comment:
Same for perspective, I thought it was based on the tournament, not the map

Reply:
For this, its either 1st, 3rd or Mixed. So surely mixed covers if both can be played on a map?

Reply:
In PUBGM (and also PC) all maps are playable on all perspectives. The perspective are only dictate manually per tournament (but technically map supports both)

Reply:
So remove the perspective altogether? As it isn't required for Maps

Reply:
remove it